### DISCUSSION TITLE
SelectList with multiple selection

### DISCUSSION THREAD

**Discussion by Lasoty** [Megabit/Blazorise#3504] [Ideas]
I know there is a **Select** component with the '**Multiple**' option. Unfortunately, it does not dynamically render **SelectItems**. I checked both @foreach and **Repeater**, it doesn't work. Unfortunately, the **SelectList** component does not have the '**Multiple**' option.

**Comment by stsrki** [comment]
This feature probably went under our radar. Shouldn't be too hard to implement I think.
